util/network/src/lib.rs

```diff
@@ -54,6 +54,9 @@
 //! }
 //! ```
 
+//TODO: use Poll from mio
+#![allow(deprecated)]
```

Comment:
you have to mark every `use::mio::<DeprecatedItem>` with this `#[allow(deprecated)]` I believe

but let me double-check